lib/katello/tasks/upgrades/4.0/remove_ostree_content.rake

```diff
@@ -0,0 +1,16 @@
+namespace :katello do
+  namespace :upgrades do
+    namespace '4.0' do
+      desc "Removes ostree content from ."
```

Comment:
I was confused by the decription "Removes ostree content" but it doesn't mention puppet content. Is that intentional?

Reply:
yes, i will fix that

Reply:
updated!